config/_default/menus.yaml

```
@@ -35,10 +38,10 @@ main:
     name: Background
     url: 'background'
     weight: 3
-  - parent: about-main
-    indentifier: about-blogs
-    name: Blogs
-    url: 'post'
+  # - parent: about-main
+  #   indentifier: about-blogs
+  #   name: Blogs
+  #   url: 'post'
     weight: 4
```

YAMLlint (1.35.1)

[error] duplication of key "weight" in mapping

(key-duplicates)